pwa/app/tailwind.css

```diff
@@ -71,4 +71,15 @@
   a {
     @apply text-blue-800 visited:text-purple-700 hover:underline;
   }
+
+  /* Create small blue buttons */
+  .btn {
+    @apply rounded px-4 py-2 font-bold;
+  }
+  .btn-blue {
+    @apply bg-blue-500 text-white;
```

Comment:
is there some constant for the TBA BLUE or should i just use the hexcode here

Reply:
The prod navbar uses `#36469c`. You can add custom colors to Tailwind by editing `tailwind.config.ts`: https://tailwindcss.com/docs/customizing-colors

Specifically we'd want to add something like:

```ts
const config = {
  // ...
  theme: {
    colors: {
      'tba': {
        DEFAULT: '#36469c',
      },
      // ...
    },
  },
}
```

Reply:
I'll hold off on this since other PRs also have buttons that should be colored, i think we can address coloring later